0110/streams/index.html

```diff
@@ -17,20 +17,20 @@
 </script>
 <script id="streams-template" type="text/x-handlebars-template">
   <h1>Kafka Streams API</h1>
-     <div class="sub-nav-sticky">
+       <div class="sub-nav-sticky">
       <div class="sticky-top">
         <div style="height:35px">
-          <a  class="active-menu-item" href="/{{version}}/documentation/streams/">Introduction</a>
-          <a href="/{{version}}/documentation/streams/developer-guide">Developer Guide</a>
+          <a  class="active-menu-item" href="#">Introduction</a>
+          <a href="/{{version}}/documentation/streams/developer-guide">Developers Guide</a>
```

Comment:
This seems a regression to me.

Reply:
Oh, good catch.  Yes, should be "Developer", not "Developers".